packages/api-file-manager/package.json

```diff
@@ -66,7 +66,8 @@
     ],
     "ignore": {
       "src": [
-        "sharp"
+        "sharp",
+        "stream"
```

Comment:
Let's move the `stream` dep to a global config, since it's a node built-in library. See `.adiorc.js:24`